Hairtrigger-Bunnies/nodejs-bangazon-site
# Dr feature

Changed:
-added ability for seller to delete a product
- 

Because:
-it is a requirement for a feature tickect

To test:
(steps to test -- include everything!)
1. add a product go to http://localhost:8080/product/add or whatever port you use and add product
2.Then click delete link 
3.To check try going back to ex: http://localhost:8080/product/30 30 represents the product id so put in the product id of the product you deleted and you should get a cannot read property error
